internal/server/audit/types.go

```diff
@@ -0,0 +1,152 @@
+package audit
+
+import (
+	"go.flipt.io/flipt/rpc/flipt"
+)
+
+// All types in this file represent an audit representation of the Flipt type that we will send to
+// the different sinks.
+
+type Flag struct {
+	Key          string `json:"key"`
+	Name         string `json:"name"`
+	Description  string `json:"description"`
+	Enabled      bool   `json:"enabled"`
+	NamespaceKey string `json:"namespace_key"`
+}
+
+func NewFlag(f *flipt.Flag) *Flag {
+	return &Flag{
+		Key:          f.Key,
+		Name:         f.Name,
+		Description:  f.Description,
+		Enabled:      f.Enabled,
+		NamespaceKey: f.NamespaceKey,
+	}
+}
+
+type Variant struct {
+	Id           string `json:"id"`
+	FlagKey      string `json:"flag_key"`
+	Key          string `json:"key"`
+	Name         string `json:"name"`
+	Description  string `json:"description"`
+	Attachment   string `json:"attachment"`
+	NamespaceKey string `json:"namespace_key"`
+}
+
+func NewVariant(v *flipt.Variant) *Variant {
+	return &Variant{
```

Comment:
could we add some very simple tests for these `NewX` funcs? Would bump up our test coverage a bit.

Reply:
@markphelps Doing that now!